pallets/gear-debug/src/lib.rs

```diff
@@ -98,7 +98,20 @@ pub mod pallet {
                 .field("static_pages", &self.static_pages)
                 .field(
                     "persistent_pages",
-                    &self.persistent_pages.keys().cloned().collect::<Vec<_>>(),
+                    &self
+                        .persistent_pages
+                        .iter()
+                        .map(|(page, data)| {
+                            // Prints only bytes which is not null
+                            let mut no_zero_bytes = BTreeMap::new();
+                            for (byte_idx, byte_value) in data.iter().enumerate() {
+                                if *byte_value != 0 {
+                                    no_zero_bytes.insert(byte_idx, *byte_value);
+                                }
+                            }
+                            (*page, no_zero_bytes)
+                        })
+                        .collect::<BTreeMap<PageNumber, BTreeMap<usize, u8>>>(),
```

Comment:
1. better to replace back usize with page_number to improve readability in logs
2. why do we need this spam with non-zero bytes?
3. it's easier to call `data.iter().filter( **check** ).collect()`

Reply:
1. No
2. Because in most cases data in wasm page is zero,  so in most cases this will looks good.
3. May be I will check it

Reply:
3th is done